src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Configuration/ICustomIdentityConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Xml;
+
+namespace CoreWCF.IdentityModel.Configuration
+{
+    /// <summary>
+    /// Types that implement ICustomIdentityConfiguration can load custom configuration
+    /// </summary>
+    public interface ICustomIdentityConfiguration
+    {
+        /// <summary>
+        /// Override LoadCustomConfiguration to provide custom handling of configuration elements
+        /// </summary>
+        /// <param name="nodeList">Xml Nodes which contain custom configuration</param>
+        void LoadCustomConfiguration(XmlNodeList nodeList);
+    }
```

Comment:
I don't think this api is used by any of this code. It's consumed by the class TypeResolveHelper which isn't in CoreWCF. You should be able to get rid of this interface and all the implementations of this method which just throw.